clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -809,4 +809,7 @@ def warn_android_unversioned_fallback : Warning<
 
 def err_drv_triple_version_invalid : Error<
   "version '%0' in target triple '%1' is invalid">;
+
+def warn_missing_include_dirs : Warning<
+  "the included directory %0 is missing">, InGroup<MissingIncludeDirs>, DefaultIgnore;
```

Comment:
nit: let's make this a little shorter and more precise by copying the sysroot warning from the test, how about: "no such include directory: ..."